ptp/c4u/clock/clock_test.go

```diff
@@ -18,34 +18,92 @@ package clock
 
 import (
 	"testing"
+	"time"
 
+	osc "github.com/facebook/time/oscillatord"
 	ptp "github.com/facebook/time/ptp/protocol"
 	"github.com/stretchr/testify/require"
 )
 
 func TestWorst(t *testing.T) {
```

Comment:
Can you please also add a test on larger dataset to demonstrate the sanity of variance. Something like in here https://github.com/facebook/time/pull/146/files#